packages/baggage-span-processor/src/index.ts

```diff
@@ -14,5 +14,5 @@
  * limitations under the License.
  */
 
-export * from './baggage-span-processor';
-export * from './types';
+export { BaggageSpanProcessor } from './baggage-span-processor';
+export { ALLOW_ALL_BAGGAGE_KEYS, BaggageKeyPredicate } from './types';
```

Comment:
note to reviewer: https://github.com/open-telemetry/opentelemetry-js/issues/4186